prep-javascript-variables/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    fullName = 'Alvaro Diaz'
+    console.log(fullName);
+
+    numberOfPets = '2'
+    console.log(numberOfPets);
+
+    likesVegetables = true;
+    console.log(likesVegetables);
```

Comment:
Hi Alvaro, be sure to use the `var` keyword when declaring variables.

```js
var numberOfPets = 2;
```

Reply:
https://github.com/alvarodiazjr/prep-0422-code-solutions/pull/10/commits/b145a79948cd82953d1f8f0c3914c9e796f07b99